client/src/js/filters/bytes.js

```diff
@@ -0,0 +1,16 @@
+angular.module('bhima.filters')
+.filter('bytes', BytesFilter);
+
+/**
+ * Bytes Filter
+ * Format bytes size to human readable format 
+ */
+function BytesFilter() {
+	return function(bytes, precision) {
+		if (isNaN(parseFloat(bytes)) || !isFinite(bytes)) return '-';
+		if (typeof precision === 'undefined') precision = 1;
+		var units = ['bytes', 'kB', 'MB', 'GB', 'TB', 'PB'],
```

Comment:
Is it ever useful to see a file size in bytes?  If so, this can remain, but I think most files will be greater than `kb`.

Also, should this be translated?  I know that in French, `Kb` --> `Ko`, `Gb` --> `Go`, etc.  Should these be translation keys instead?